### ISSUE TITLE
AvoidTooShortNames false negative

### ISSUE THREAD

**Issue by knocte** [fsprojects/FSharpLint#825]
Taken from FSharpLint code itself (src/FSharpLint.Core/Framework/Ast.fs):

```
    let rec private flattenFuncExpr flattened exprToFlatten =
        match exprToFlatten with
        | SynExpr.App(_, _, x, y, _) ->
            ...
```
